transparency/config_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) The go-boot authors. All Rights Reserved.
+//
+// Use of this source code is governed by the license
+// that can be found in the LICENSE file.
+
+// Package transparency implements an interface to the
+// boot-transparency library functions to ease boot bundle
+// validation.
+package transparency
+
+import (
+	"testing"
+
+	"github.com/usbarmory/boot-transparency/artifact"
+)
+
+func TestPath(t *testing.T) {
+	kernelHash := "70672136965536be27980489b0388d864c96c74efd73d21432d0bcf10f9269f3"
```

Comment:
These should be constants defined earlier outside the function (also to avoid repetition with later functions).